code/modules/client/preference/preferences_toggles.dm

```diff
@@ -340,3 +340,11 @@
 	prefs.toggles2 ^= PREFTOGGLE_2_REVERB_DISABLE
 	prefs.save_preferences(src)
 	to_chat(src, "You will [(prefs.toggles2 & PREFTOGGLE_2_REVERB_DISABLE) ? "no longer" : "now"] get reverb on ingame sounds.")
+
+/client/verb/toggle_forced_white_runechat()
+	set name = "Toggle Runechat Colour Forcing"
+	set category = "Preferences"
+	set desc = "Toggles forcing your runechat colour to white"
```

Comment:
Not quite sure if this falls under the "You should use American spelling in code" rule, since its user facing, but I'll leave this here anyway.
```suggestion
	set name = "Toggle Runechat Color Forcing"
	set category = "Preferences"
	set desc = "Toggles forcing your runechat color to white"
```